Add materialized view materialization (#260)
starburstdata/dbt-trino: README.md

}}
```

##### Materialized view

> Adapter supports [materialized views](https://trino.io/docs/current/sql/create-materialized-view.html).
> At every subsequent `dbt run` command, materialized view would be [refreshed](https://trino.io/docs/current/sql/refresh-materialized-view.html).

Reviewer: ```suggestion
The adapter also supports [materialized views](https://trino.io/docs/current/sql/create-materialized-view.html).
At every subsequent `dbt run` command, the materialized view is [refreshed](https://trino.io/docs/current/sql/refresh-materialized-view.html).
```

> This materialization supports [full_refresh](https://docs.getdbt.com/reference/resource-configs/full_refresh) config and flag.
> Whenever you want to rebuild your materialized view, e.g. when changing underlying SQL query, run `dbt run --full-refresh`.

Reviewer: ```suggestion
This materialization supports the [full_refresh](https://docs.getdbt.com/reference/resource-configs/full_refresh) config and flag.
Whenever you want to rebuild your materialized view, e.g. when changing underlying SQL query, run `dbt run --full-refresh`.
```

In model add:
```jinja2
{{

>   config(

Reviewer: Also add the properties syntax when implemented here.

}}
```

##### Materialized view

The adapter also supports [materialized views](https://trino.io/docs/current/sql/create-materialized-view.html).
At every subsequent `dbt run` command, the materialized view is [refreshed](https://trino.io/docs/current/sql/refresh-materialized-view.html).

> You could define properties for the materialized view creation by `properties` config.

Reviewer: ```suggestion
You can also define custom properties for the materialized view through the `properties` config.
```
